Update DC/OS CLI configuration settings

jonas opened the pull request:
## What are your changes?
Update documentation for the CLI based on the schema found here:
- https://github.com/dcos/dcos-cli/blob/3c95b2b80774d9d7b2888d7c88f4cdd42a405fe9/dcos/data/config-schema/core.json

## What type of changes does your doc introduce?
- [x] Bug fix
- [ ] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [ ] High
- [x] Medium

## I have completed these items:
- [x] I have tested all commands and code snippets.
- [ ] I have built locally and tested for broken links and formatting.
- [ ] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [ ] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping `@emanic`, `@sascala`, or `@joel-hamill` for reviewing pull request changes.


## If you included a comment with your commit, it appears here:

tamarrow commented on 1.8/usage/cli/index.md:
```diff
@@ -87,45 +87,45 @@ $ dcos config set core.dcos_url 52.36.102.191
     
 
-**email** Your email address. This is set by default during installation. For example, to reset your email address:
+**dcos_acs_token** The token generated by authenticating to DC/OS with ACS. For example, to change the ACS token:
```
Lets use `Access Control Service (ACS) for the first reference

tamarrow replied:
Instead of `to change the ACS token`: `To set the ACS token, run `dcos auth login`

tamarrow commented:
thanks @jonas just a few comments

jonas commented:
@tamarrow Updated to include the suggested improvement and rebased against latest master. I've also updated the CLI output snippet to mirror version 0.4.16 of the CLI.

tamarrow commented:
thanks @jonas LGTM

joel-hamill commented:
@jonas can you make this same change for the /1.9 doc version too? thanks!

jonas commented:
Sorry @joel-hamill, this slipped. I see you updated for 1.9.

joel-hamill commented:
@jonas np, thanks for your contribution!
